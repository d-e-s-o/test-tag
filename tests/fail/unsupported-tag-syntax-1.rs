// Copyright (C) 2024 Daniel Mueller <deso@posteo.net>
// SPDX-License-Identifier: (Apache-2.0 OR MIT)

/// Unsupported key-value syntax.
#[test_tag::tag(tag1 = "foobar")]
#[test]
fn cant_compile() {}
