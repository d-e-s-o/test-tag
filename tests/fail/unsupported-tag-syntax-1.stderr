error: expected `,`
 --> tests/fail/unsupported-tag-syntax-1.rs:5:22
  |
5 | #[test_tag::tag(tag1 = "foobar")]
  |                      ^

error[E0601]: `main` function not found in crate `$CRATE`
 --> tests/fail/unsupported-tag-syntax-1.rs:7:21
  |
7 | fn cant_compile() {}
  |                     ^ consider adding a `main` function to `$DIR/tests/fail/unsupported-tag-syntax-1.rs`
